src/current/_includes/releases/v24.2/feature-highlights-observability.html

```diff
@@ -0,0 +1,41 @@
+<table>
+ <thead>
+  <tr>
+   <th class="center-align" colspan="1" rowspan="2">Feature</th>
+   <th class="center-align" colspan="4" rowspan="1">Availability</th>
+  </tr>
+  <tr>
+   <th colspan="1" rowspan="1">Ver.</th>
+   <th colspan="1" rowspan="1" style="white-space: nowrap;">Self-Hosted</th>
+   <th colspan="1" rowspan="1">Dedicated</th>
+   <th colspan="1" rowspan="1">Serverless</th>
+  </tr>
+ </thead>
+ <tbody>
+  <tr>
+   <td>
+    <p class="feature-summary">Azure Dedicated clusters integrate with Azure Monitor for metrics and logs (limited access)</p>
+    <p class="feature-description">
+     <a href="https://www.cockroachlabs.com/docs/cockroachcloud/export-logs?filters=azure-monitor-log-export">Exporting logs</a> to Azure Monitor Logs and <a href="https://www.cockroachlabs.com/docs/cockroachcloud/export-metrics?filters=azure-monitor-metrics-export">exporting metrics</a> to Azure Monitor Metrics and Prometheus-compatible metric sinks from your CockroachDB Dedicated cluster hosted on Azure is in limited access.
```

Comment:
She is on PTO but this is definitely the text she provided (for context see [the sheet](https://docs.google.com/spreadsheets/d/1_ecmsBTd-uxSLeG05gDiCZ_8mAg4lXMCmYKeRR4C--g/edit?gid=1382804691#gid=1382804691), it seems pretty clear they wanted limited access since it's also mentioned in another column).

Def added the link, thank you for that!